src/client/mod.rs

```diff
@@ -288,6 +288,21 @@ impl Client {
         }
     }
 
+    /// A helper method to start a `ChangeStream`. Used by watch methods in client, coll, and db.
+    pub(crate) async fn start_change_stream(
+        &self,
+        pipeline: impl IntoIterator<Item = Document>,
+        mut options: Option<ChangeStreamOptions>,
+        target: ChangeStreamTarget,
+    ) -> Result<ChangeStream> {
+        let pipeline: Vec<Document> = pipeline.into_iter().collect();
+        resolve_options!(self, options, [selection_criteria]);
```

Comment:
We should also accept a read concern here via the options like we do for other read operations. According to the spec, we must document that only the default (i.e. no read concern) or "majority" read concern levels are supported in 3.6 and 4.0, and that starting in 4.2, any read concern can be specified.

From the spec:
>The initial implementation of change streams on the server requires a “majority” read concern or no read concern. Drivers MUST document this requirement. Drivers SHALL NOT throw an exception if any other read concern is specified, but instead should depend on the server to return an error.

Reply:
added

Reply:
The requisite documentation still needs to be added.

Reply:
added!